src/extra/libs/gif/lv_gif.c

```
@@ -119,35 +126,76 @@ static void lv_gif_destructor(const lv_obj_class_t * class_p, lv_obj_t * obj)
 
 static void next_frame_task_cb(lv_timer_t * t)
 {
     lv_obj_t * obj = t->user_data;
+    if(obj == NULL) {
+        return;
+    }
+    
     lv_gif_t * gifobj = (lv_gif_t *) obj;
+    
+    if(gifobj == NULL || gifobj->gif == NULL || gifobj->imgdsc.data == NULL) {
+        return;
```

Cppcheck (2.19.0)

[style] The function 'lv_slider_get_max_value' is never used.

(unusedFunction)